productcat/src/main/java/com/visenze/productcat/android/util/ProductCatUIDManager.java

```diff
@@ -22,7 +22,12 @@
     private static SharedPreferences preference;
 
     public static void initPref(final Context context) {
-        preference =  context.getSharedPreferences(PREFS_NAME, Context.MODE_PRIVATE);
+
+        String pref_file = context.getString(R.string.preference_file);
+        preference = context.getSharedPreferences(pref_file, Context.MODE_PRIVATE);
+        // PS-303. add uid by default.
+        String uniqueID = UUID.randomUUID().toString();
+        setUid(uniqueID);
```

Comment:
also can you remove storeUidIfNeeded method? We no longer process UID in the cookie. It must be passed via query parameter.

Reply:
deleted already